Assets/HoloToolkit-Preview/QRTracker/Plugins/WSA/arm/QRCodesTrackerPlugin.dll.meta

```diff
@@ -0,0 +1,30 @@
+fileFormatVersion: 2
+guid: 980931158bfdfdd47ab8de3098f68f38
```

Comment:
Is the source for this dll availible or in the native MRTK repo?

Or is this something that will be replaced by native windows api calls?

Reply:
it is currently not available. this is an experimental feature. when any final form is ready, the code will be updated

Reply:
This dll will be copied from windows latest builds ( they are built in RS6 windows builds) to the MRTK on very slow cadence ( any bugs).